content/en/Getting started/Assets/_index.md

```diff
@@ -10,15 +10,15 @@ description: >
 Help our pentesters test your [assets](../glossary#asset) faster.
 {{% /pageinfo %}}
 
-When you see a **Let's get started!** screen, select **Create a new pentest for a new asset**.
-The other option is beyond the scope of our Getting Started guide.
+The **Let's get started!** screen includes two options:
 
-When you set up a pentest through the UI, you're going through the following stages of our pentest wizard:
+- **Create a new pentest from an existing asset**
+  - This option opens a drop-down text box. Use it to select from assets that you've
+    previously created. It populates the Asset screen with available information.
+- **Create a new pentest for a new asset**.
 
-- Define Your Assets
-- Create Pentest Objectives
-- Add Details
-- Plan the Pentest
+<!-- For content, see https://github.com/cobalthq/cobalt-product-public-docs/blob/main/layouts/shortcodes/big-pic-UI-steps.md -->
```

Comment:
Until the PR is merged, you'll see this file in the following directory: https://github.com/cobalthq/cobalt-product-public-docs/tree/DOCS-70_incorporate_5steps_article_from_zendesk/layouts/shortcodes